code/cmdline/cmdline.cpp

```diff
@@ -510,6 +512,7 @@ int Cmdline_reparse_mainhall = 0;
 bool Cmdline_profile_write_file = false;
 bool Cmdline_no_unfocus_pause = false;
 bool Cmdline_benchmark_mode = false;
+char *Cmdline_pilot = nullptr;
```

Comment:
Please use `SCP_string` to avoid issues with memory management.

Reply:
Are you certain about this? Other flags in this file use `char*` without exception, `SCP_string` is not used in any exported flag variable.

Reply:
Huh, I thought that was changed at some point. I guess char is fine then but it should still be `const char*`.

Reply:
It's not `const` anywhere else either, but I agree, it should be.